v3/src/components/app.tsx

```diff
@@ -1,21 +1,22 @@
 import React from "react"
 import { CaseTable } from "./case-table/case-table"
-import { Container } from "./container"
-import { DataSummary } from "./data-summary"
-import { gDataBroker } from "../data-model/data-broker"
-import { DataSet, toCanonical } from "../data-model/data-set"
-import { Text } from "./text"
-import { useDropHandler } from "../hooks/use-drop-handler"
-import { useSampleText } from "../hooks/use-sample-text"
+import {Container} from "./container"
+import {DataSummary} from "./data-summary"
+import {gDataBroker} from "../data-model/data-broker"
+import {DataSet, toCanonical} from "../data-model/data-set"
+import {Text} from "./text"
+import {useDropHandler} from "../hooks/use-drop-handler"
+import {useSampleText} from "../hooks/use-sample-text"
 import Icon from "../assets/concord.png"
```

Comment:
I'm guessing the IntelliJ auto-formatter is removing the spaces before/after the braces. I'd recomment turning that off as I don't think we want to mandate one way or the other and automatic white-space only changes are annoying on review.